api/src/server.ts

```diff
@@ -1,35 +1,35 @@
+import * as Sentry from '@sentry/node';
```

Comment:
Why are stuff moved around? Makes it hard to read the diff

Reply:
No idea, must have been from prettier or lint or some auto formatter. Nothing purposeful

Reply:
Hmmm, yeah, looks like some sort of import sorting got run here.